vkrunner/vr-script.c

```diff
@@ -1592,6 +1604,27 @@ process_test_line(struct load_state *data)
                 p = command_start;
         }
 
+        if (looking_at(&p, "tolerance ")) {
+                const char *p_backup = p;
+                if (parse_doubles(&p, data->tolerance, 4, NULL)) {
+                        data->tolerance_in_percent = false;
+                        return true;
+                }
+                p = p_backup;
+                if (parse_doubles(&p, data->tolerance, 4, "%")) {
```

Comment:
This is neat but I guess it will mean the final value doesn’t have to have a percent sign. Ie, this would be valid:

> tolerance 1% 1% 1% 1

I’m not sure if it really matters or not.